lighthouse-cli/test/cli/cli-flags-test.js

```diff
@@ -10,24 +10,21 @@
 import {strict as assert} from 'assert';
 import fs from 'fs';
 
-import yargs from 'yargs';
-
-import {getFlags} from '../../cli-flags.js';
+import {getFlags, getYargsParser} from '../../cli-flags.js';
 import {LH_ROOT} from '../../../root.js';
 
 describe('CLI flags', function() {
   it('all options should have descriptions', () => {
-    getFlags('chrome://version');
```

Comment:
with jest `import yargs from 'yargs` here and in the test file results in `yargs.getGroups()` working in the test file. That function is added to the module dynamically after being called, which is why this test did this seemingly no-op call to `getFlags`.

But, that property is only added to the module object in jest because ... reasons? The modules aren't really es modules, I guess, and es module objects are read-only, IIRC.

Anyhow, the `getGroups` method is _also_ added to the yargs parser object, so to get to that I split up the implementation across two functions.